README.md

```diff
@@ -84,7 +84,7 @@ Testing for the ier-frontend project is via the sbt command line as follows...
 
  3. Or just `./sbt test`
 
- **NB : Make sure you follow the Bootstrap Step 5 above to make the /var/log/ier/ folder locally wriateable.**
+ **NB : Make sure you follow the Bootstrap Step 5 above to make the /var/log/ier/ folder locally writable.**
```

Comment:
I'd have gone with `writeable` but apparently either is OK (and your version more common).